pkg/shell/shell.go

```diff
@@ -114,9 +115,14 @@ func (sp ShellProxy) ConnectFunc(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	sshUsername := vm.Spec.SshUsername
+	if len(sshUsername) < 1 {
```

Comment:
Just curious, why are you doing `< 1` instead of `== 0`?

Reply:
Just what my mind did in deference to doing `> 0` in other checks lol.